content/ja/docs/languages/java/intro.md

```diff
@@ -0,0 +1,130 @@
+---
+title: OpenTelemetry Java入門
+description: OpenTelemetry Javaエコシステム入門
+weight: 9
+default_lang_commit: beb85b4f56de76aa8a8d6e96cd7528396476f95a
+---
+
+OpenTelemetry JavaはJavaエコシステム向けのOpenTelemetryオブザーバビリティツールのセットです。
+大まかに言えば、API、SDK、および計装で構成されています。
+
+このページでは、概念的な[概要](#overview)、[ドキュメントのナビゲート](#navigating-the-docs)ガイド、リリースとアーティファクトに関する主要な詳細を含む[リポジトリ](#repositories)のリストを使用して、エコシステムを紹介します。
+
+## 概要 {#overview}
+
+APIは、主要なオブザーバビリティシグナル全体でテレメトリを記録するためのクラスとインターフェースのセットです。
+複数の実装をサポートし、低オーバーヘッドのミニマリストNoop（「ノーオプ」と発音）とSDKリファレンス実装がすぐに使用できるように提供されています。
+ライブラリ、フレームワーク、および計装を追加しようとしているアプリケーション所有者が直接依存関係として取得するように設計されています。
+強力な後方互換性の保証、ゼロ推移的依存関係が付属し、Java 8以降をサポートしています。
+
+SDKはAPIの組み込みリファレンス実装であり、計装APIコールによって生成されたテレメトリを処理およびエクスポートします。
+SDKを適切に処理およびエクスポートするように構成することは、OpenTelemetryをアプリケーションに統合するための重要なステップです。
+SDKには、自動構成とプログラマティック構成のオプションがあります。
```

Comment:
```suggestion
SDKには、自動構成とプログラムを書いて構成するオプションがあります。
```

Reply:
Thank you for your review.
I fixed in a bellow commit.

https://github.com/open-telemetry/opentelemetry.io/pull/7057/commits/5c102c6976e0a67c24998363082c5e332b9a78d8